packages/@snowpack/app-template-svelte/snowpack.config.json

```diff
@@ -1,5 +1 @@
-{
-  "extends": "@snowpack/app-scripts-svelte",
-  "scripts": {},
-  "plugins": []
-}
+{"extends":"@snowpack/app-scripts-svelte","scripts":{},"plugins":[],"buildOptions":{"minify":false}}
```

Comment:
Might want to revert this. This happens when you run `yarn test` locally and commit a file while the test is running. We modify the `snowpack.config.json` to add `{ minify: false }` in the test because there’s not a CLI flag to do it (and without `minify: false` the snapshots would change all the time because esbuild’s minification changes across versions).

Reply:
done!